### PR TITLE
Foyer docs udpate

### PR DESCRIPTION
**Pull request by daico007**
### PR Summary:
Update Foyer docs with current mbuild organization, refer to #265. 

### PR Checklist
------------
 - [ ] Includes appropriate unit test(s)
 - [ ] Appropriate docstring(s) are added/updated
 - [ ] Code is (approximately) PEP8 compliant
 - [ ] Issue(s) raised/addressed?

### CODE REVIEW

**Comment by umesh-timalsina** [comment]
3ad9b18, changes the docstring on `test.utils`, adds `examples` with similar utils function. Both the `XML` files are provided as package data.

File: docs/paper_examples.rst [hunk 1]
```diff
@@ -69,22 +69,24 @@ combined into a single ``ParmEd`` ``Structure`` and saved to disk.
 .. code:: python
 
     from foyer import Forcefield
-    from foyer.tests.utils import get_fn
+    from foyer.examples.utils import get_example_file
```
Comment (ahy3nz) [inline docs/paper_examples.rst]: `example_file_path` not `get_example_file`

File: docs/paper_examples.rst [hunk 2]
```diff
@@ -69,22 +69,24 @@ combined into a single ``ParmEd`` ``Structure`` and saved to disk.
...
+    silica_substrate=Monolayer(surface=silica,chains=H(),guest_port_name="up")
     # Determine the box dimensions dictated by the silica substrate
     box=mb.Box(mins=[0, 0,max(silica.xyz[:,2])],maxs=silica.periodicity+ [0, 0, 4])
     # Fill the box with ethane
-    ethane_fluid=mb.fill_box(compound=Ethane(),n_compounds=200,box=box)
+    ethane_fluid=mb.fill_box(compound=Eth   ane(),n_compounds=200,box=box)
     # Load the forcefields
-    opls_silica=Forcefield(forcefield_files=get_fn("oplsaa_with_silica.xml"))
-    opls_alkane=Forcefield(forcefield_files=get_fn("oplsaa_alkane.xml"))
+    opls_silica=Forcefield(forcefield_files=get_example_file("oplsaa_with_silica.xml"))
```
Comment (ahy3nz) [inline docs/paper_examples.rst]: `example_file_path`

File: docs/paper_examples.rst [hunk 3]
```diff
@@ -69,22 +69,24 @@ combined into a single ``ParmEd`` ``Structure`` and saved to disk.
...
+    opls_alkane=Forcefield(forcefield_files=get_example_file("oplsaa_alkane.xml"))
```
Comment (ahy3nz) [inline docs/paper_examples.rst]: `example_file_path`

File: docs/paper_examples.rst [hunk 4]
```diff
@@ -69,22 +69,24 @@ combined into a single ``ParmEd`` ``Structure`` and saved to disk.
...
     # Create a silica substrate, capping surface oxygens with hydrogen
-    silica=mb.recipes.SilicaInterface(bulk_silica=AmorphousSilica())
-    silica_substrate=mb.recipes.Monolayer(surface=silica,chains=H(),guest_port_name="up")
+    silica=SilicaInterface(bulk_silica=AmorphousSilica())
+    silica_substrate=Monolayer(surface=silica,chains=H(),guest_port_name="up")
     # Determine the box dimensions dictated by the silica substrate
     box=mb.Box(mins=[0, 0,max(silica.xyz[:,2])],maxs=silica.periodicity+ [0, 0, 4])
     # Fill the box with ethane
-    ethane_fluid=mb.fill_box(compound=Ethane(),n_compounds=200,box=box)
+    ethane_fluid=mb.fill_box(compound=Eth   ane(),n_compounds=200,box=box)
```
Comment (ahy3nz) [inline docs/paper_examples.rst]: Weird space `Eth   ane()`?

**Comment by ahy3nz** [comment]
I did a code review, but ended up making those changes (using the right util func, weird spacing) in https://github.com/mosdef-hub/foyer/pull/266/commits/508a23dc6598d99797009d983430541522f5895c.

There was a new issue with the oplsaa_with_silicon xml because certain overrides or RBTorsions were not defined in the AtomTypes, so I attached a script (and output xml) that trims the oplsaa_with_silicon to just the relevant atomtypes, overrides, and rbtorsions. With this new xml, the systems parametrized (mostly) okay. I'll leave it to you guys to re-name or delete some of the extra files I added.

The next thing is the Osmium issue which I think is being addressed [in this mbuild issue](https://github.com/mosdef-hub/mbuild/pull/593)

**Comment by daico007** [comment]
@uppittu11 Can you upload the screenshot of that line?

**Comment by uppittu11** [comment]
image

It's the last code block of the `usage_examples.rst`
image

File: docs/usage_examples.rst [hunk 5]
```diff
@@ -64,22 +64,24 @@ operator and can be saved to Gromacs files.
 .. code:: python
 
     from foyer import Forcefield
-    from foyer.tests.utils import get_fn
+    from foyer.tests.utils import example_file_path
```
Comment (uppittu11) [inline docs/usage_examples.rst]:
```suggestion
    from foyer.examples.utils import example_file_path
```
forgot to change the directory

**Pull request merged**